src/components/ui/ProgressPanel.svelte

```diff
@@ -54,19 +54,68 @@
       default: 'View report'
     }),
     CONFORMANCE_LEVEL: $translate('WCAG.COMMON.CONFORMANCE_LEVEL'),
-    REPORT_FOR: $translate('UI.REPORT.REPORT_FOR')
+    REPORT_FOR: $translate('UI.REPORT.REPORT_FOR'),
+    REPORTED_ON: $translate('UI.REPORT.REPORTED_ON'),
+    REPORTED_ON_OF: $translate('UI.REPORT.REPORTED_ON_OF')
   };
 
-  $:  console.log('totalevaluated', totalEvaluated);
   $: conformanceTarget = $scopeStore['CONFORMANCE_TARGET'];
-  $: wgacVersion = $scopeStore['WCAG_VERSION'];
-  $: percentageEvaluated = (totalEvaluated / totalToEvaluate) * 100;
+  $: wcagVersion = $scopeStore['WCAG_VERSION'];
+  $: percentageTotalEvaluated = (totalEvaluated / totalToEvaluate) * 100;
 
   $: principles = [...new Set($wcag.map((a) => a.num.split('.')[0]))];
 
-  $: console.log($assertions);
+  $: upToWcagVersion = wcagVersion === "2.0" ?
+    ["2.0"] : ["2.0", "2.1"];
 
-  let totalEvaluated = 3;
+  $: filteredCriteria = {
+   1: $wcag.filter(item => item.num.startsWith("1.")).filter(isInScope) || {},
+   2: $wcag.filter(item => item.num.startsWith("2."))
+    .filter(isInScope) || {},
+   3: $wcag.filter(item => item.num.startsWith("3."))
+    .filter(isInScope) || {},
+   4: $wcag.filter(item => item.num.startsWith("4."))
+    .filter(isInScope) || {}
+  };
+
+  $: filteredAssertions = {
```

Comment:
Good to take these calcs to auditStore; `filteredAssertions`, `totalsPerPrinciple`

Reply:
Added to separate clean up task https://github.com/w3c/wcag-em-report-tool/issues/411